common/v2/features/Dashboard/components/WalletBreakdown.tsx

```diff
@@ -76,11 +76,6 @@ function WalletBreakdown() {
   return (
     <div className="WalletBreakdown">
       <div className="WalletBreakdown-selectWrapper">
-        <ComboBox
-          className="WalletBreakdown-selectWrapper-select"
```

Comment:
Please remove the relevant SCSS

Reply:
this is still used for the accounts dropdown actually